tpch-scripts/getStoreLatency.sh

```diff
@@ -6,6 +6,7 @@ fi
 which csvsql > /dev/null 2>&1
 if [ $? -ne 0 ]; then
 	echo "Installing CSVKIT" 
+	pip install --upgrade pip
```

Comment:
may be add a comment on why it was added. "3.5 cluster ships with old pip version which fails while any package install"